distributed/stealing.py

```diff
@@ -120,22 +119,24 @@ def steal_time_ratio(self, ts):
         if not ts.dependencies:  # no dependencies fast path
             return 0, 0
 
-        nbytes = ts.get_nbytes_deps()
-
-        transfer_time = nbytes / self.scheduler.bandwidth + LATENCY
```

Comment:
We don't even use these until the 3rd `if`. So just move them later. We could already have `return`ed before needing these.